exercises/binary-search-tree/BinarySearchTreeTest.fs

```diff
@@ -6,40 +6,40 @@ open BinarySearchTree
 [<Test>]
 let ``Data is retained`` () =
     let tree = singleton 4
-    Assert.That(tree |> value, Is.EqualTo(4))
+    Assert.That(tree |> value, Is.EqualTo(Some 4))
 
 [<Test>]
 [<Ignore("Remove to run test")>]
 let ``Inserting less`` () =
     let tree = singleton 4 |> insert 2
-    Assert.That(tree |> value, Is.EqualTo(4))
-    Assert.That(tree |> left |> Option.map value, Is.EqualTo(Some 2))
+    Assert.That(tree |> value, Is.EqualTo(Some 4))
+    Assert.That(tree |> left |> Option.bind value, Is.EqualTo(Some 2))
 
 [<Test>]
 [<Ignore("Remove to run test")>]
 let ``Inserting same`` () =
     let tree = singleton 4 |> insert 4
-    Assert.That(tree |> value, Is.EqualTo(4))
-    Assert.That(tree |> left |> Option.map value, Is.EqualTo(Some 4))
+    Assert.That(tree |> value, Is.EqualTo(Some 4))
+    Assert.That(tree |> left |> Option.bind value, Is.EqualTo(Some 4))
 
 [<Test>]
 [<Ignore("Remove to run test")>]
-let ``Inserting greater`` () =
+let ``Inserting greater()`` =
```

Comment:
This isn't consistent with the above test cases. I'm not sure the difference between before or after, but I don't see why they wouldn't be consistent.